backend/graphql/schema.graphql

```diff
@@ -33,6 +33,50 @@ input ValidateNewUserInput @validator(class: "UserInputValidator") {
     email: String
 }
 
+"""
+Input type for adding social media data to profile metadata
+"""
+input UpdateProfileMetadataSocialMediaInput @validator(class: "ProfileMetadataSocialMediaValidator") {
```

Comment:
Can we use namespaces in the class property?  I.e. would it be better to be able to make `App\Graphql\Validators\ProfileMetadata\SocialMediaValidator` to keep the validators organized?  I don't know if the class property will happily accept a namespaced class or not though.

Reply:
No, it doesn't seem to be happily accepting this namespace.